src/Core/Style.js

```diff
@@ -0,0 +1,113 @@
+import { FEATURE_TYPES } from 'Core/Feature';
+
+function rgba2rgb(orig) {
+    if (orig && orig.includes('rgba')) {
+        const rgb = orig.replace(/\s/g, '').match(/^rgba?\((\d+),(\d+),(\d+),(\d\.\d)/i);
+        return (rgb && rgb.length === 5) ? { color: `rgb(${rgb[1]}, ${rgb[2]}, ${rgb[3]})`, opacity: rgb[4] } : { style: orig, opacity: 1.0 };
+    } else if (orig && orig.includes('hsla')) {
+        const hsl = orig.replace(/\s/g, '').match(/^hsla?\((\d+),(\d+)%,(\d+)%,(\d\.\d)/i);
+        return (hsl && hsl.length === 5) ? { color: `rgb(${hsl[1]}, ${hsl[2]}, ${hsl[3]})`, opacity: hsl[4] } : { style: orig, opacity: 1.0 };
+    } else {
+        return { color: orig, opacity: 1.0 };
+    }
+}
+
+/**
+ * Style defines {@link Feature} style.
+ */
+class Style {
+    /**
+     * @constructor
+     * @param {object} [params]
+     * @param {object} [params.fill] fill style.
+     * @param {string} [params.fill.color] fill color string css.
+     * @param {number} [params.fill.opacity] fill opacity.
+     * @param {THREE.Vector3} [params.fill.color] fill color.
+     * @param {object} [params.stroke] stroke style.
+     * @param {THREE.Vector3} [params.stroke.color] stroke
+     * @param {string} [params.stroke.color] stroke color string css.
+     * @param {number} [params.stroke.opacity] stroke opacity.
+     * @param {number} [params.stroke.width] stroke line width.
+     * @param {object} [params.point] point style.
+     * @param {THREE.Vector3} [params.point.color] point
+     * @param {string} [params.point.color] point color string css.
+     * @param {number} [params.point.opacity] point opacity.
+     * @param {number} [params.point.radius] point line radius
+     */
+    constructor(params = {}) {
```

Comment:
Some parameters are in double here. Also you could simply write that `params` is an object that can contain all properties of `Style`, and move the list of parameters above, something like the documentation of `Source`.

Reply:
You still need to change the documentation here.